control-plane/tests/test_cross_boundary_negative_e2e_validation.py

```
@@ -264,6 +264,207 @@ def test_soft_write_receipt_mismatch_stays_non_authoritative_in_case_detail_surf
+        current_case = store.get(CaseRecord, promoted_case.case_id)
+        self.assertIsNotNone(current_case)
+        self.assertEqual(current_case.lifecycle_state, baseline_case.lifecycle_state)
+        self.assertEqual(current_case.evidence_ids, baseline_case.evidence_ids)
+        self.assertEqual(
+            {record.evidence_id for record in store.list(EvidenceRecord)},
+            baseline_evidence_ids,
+        )
+
+    def test_expired_delegation_fails_closed_before_action_execution_dispatch(self) -> None:
+        phase28_helper = phase28_tests.Phase28EndpointEvidencePackValidationTests()
+        store, service, promoted_case, anchor_evidence_id, reviewed_at = (
```

Ruff (0.15.12)

[warning] Unpacked variable `reviewed_at` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)